<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertBookWidget</class>
 <widget class="QWidget" name="InsertBookWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>书名：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="booknameLineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>出版社：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QLineEdit" name="publisherLineEdit"/>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>类别：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QComboBox" name="catagoryComboBox"/>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="insertPushButton">
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>数量：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="numberSpinBox"/>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>作者：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLineEdit" name="authorLineEdit"/>
   </item>
   <item row="2" column="0" colspan="8">
    <widget class="BookList" name="bookList" native="true"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>出版年份:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QSpinBox" name="publishYearSpinBox">
     <property name="minimum">
      <number>1800</number>
     </property>
     <property name="maximum">
      <number>2300</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BookList</class>
   <extends>QWidget</extends>
   <header location="global">booklist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
